src/extensions/nuget/Microsoft.DotNet.UpgradeAssistant.Extensions.NuGet/PackageLoader.cs

```diff
@@ -122,19 +123,24 @@ private async Task<IEnumerable<NuGetReference>> SearchByNameAsync(string name, I
                 }
             }
 
-            return FilterSearchResults(name, results, tfms, currentVersion, options.LatestMinorAndBuildOnly);
+            // Switch to an IEnumerable here because consumers of this API 
```

Comment:
I think this one's breaking the build.
```suggestion
            // Switch to an IEnumerable here because consumers of this API
```